i18n.php

```diff
@@ -921,7 +921,7 @@
     'but' => 'Nhưng',
     'examples' => 'Dữ liệu',
     'feature' => 'Tính năng',
-    'given' => 'Biết|Cho',
+    'given' => '*|Biết|Cho',
```

Comment:
change to this?https://github.com/cucumber/gherkin/blob/master/ruby/lib/gherkin/gherkin-languages.json